src/folder/Normal.js

```diff
@@ -1,28 +1,34 @@
 import { Country, State, City } from "country-state-city";
-import React from "react";
-import Dropdown from "react-dropdown";
-import PhoneInput from "react-phone-number-input";
-import "./signup.css";
-
-
-const allCountry = Country.getAllCountries();
-
-const countryCodes = require("country-codes-list");
-console.log(allCountry);
+import React, { useState } from "react";
 
+import "./signup.css";
 
+const CountryAndStateComponent = () => {
+  let countryIsoCode = "";
+  const allCountry = Country.getAllCountries();
+  const [countryCode, setCountryCode] = useState("");
 
-console.log("ddd", countryCodes);
-const CountryAndStateComponent = ({ countryCode = "IN" }) => {
-  const [state, setState] = React.useState([]);
-  const [value, setValue] = React.useState();
+  const [states, setStates] = React.useState([]);
+  const [cities, setCities] = React.useState([]);
 
   const handleCountry = (e) => {
-    const countrydata = e.target.value;
+    countryIsoCode = e.target.value;
+    setCountryCode(countryIsoCode);
+    const countryStates = State.getStatesOfCountry(countryIsoCode);
+    console.log("EEEE", countryStates);
+    setStates(countryStates);
+
+    console.log("jj", countryStates);
   };
 
-  const get = (e) => {
-    console.log(e);
+  const handleState = (e) => {
+    const stateIsoCode = e.target.value;
+    console.log("DDDd", stateIsoCode, "ggg", countryIsoCode);
```

Comment:
remove console

Reply:
Remove Console and Normal component done.